lab.R

```diff
@@ -16,13 +16,16 @@
 #' We'll be accessing the dataset using the `AmesHousing` package.  (Strictly speaking, we'll be using a CSV that I generated from one version in the `AmesHousing` package, because it gives us a chance to learn a few things about factors.)  It'll be useful to have the documentation for `AmesHousing::ames_raw` open, because it gives short descriptions of the many variables in the dataset:  <https://cran.r-project.org/web/packages/AmesHousing/AmesHousing.pdf>.  You can find a full codebook for the dataset at <http://jse.amstat.org/v19n3/decock/DataDocumentation.txt>. 
 #' 
 
+#testing
+
 ## Setup
 ## **IMPORTANT**: Add all dependencies to `DESCRIPTION`
 library(tidyverse)
 library(skimr)
 library(visdat)
+library(dplyr)
```

Comment:
Loaded as part of the tidyverse
```suggestion
```

Reply:
It gave me an error when I didnt include it